data_structures/binary_tree.cpp

```diff
@@ -0,0 +1,83 @@
+// Given a binary tree, print the inorder traversal of the binary tree
+#include <vector>
+#include <iostream>
+#include <stack>
+using namespace std;
+
+struct node
```

Comment:
`Node` instead of `node`.
Structs and classes should always have their first letter capitalized.